imports/api/logic/grad-checker/faculty/computing/computer-science/AY-1516/AY1516GradCheckerHandler.js

```diff
@@ -292,12 +244,98 @@ const UIFormatConversion = function UIFormatConversion(name, markedModules, isFu
 }
 
 /**
-* retrieves an object of UI formatted graduation requirements for a focus area
+* retrieves an object of UI formatted marked graduation requirement
+*  @param {object}  parent grad requirement object
+*  @param {object}  an object containing a list of modules in the format of modueCode:bool
+*  @return {{objects}}  UI formatted list for one set of marked requirements
+*
+*/
+const createUIFormat = function createUIFormat(tempGradRequirement, modules)  {
```

Comment:
Good to abstract away the UI format from the actual implementation of other functions in this grad requirements module